Data-Storage/hint_messages.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "en_us": "Press {keybind:Share Item} on a gear item to share it in chat or save it."
+  },
+  {
+    "en_us": "In Overlay Management you can move, resize and change alignments of all overlays."
+  },
+  {
+    "en_us": "Every single feature in Wynntils can be toggled on or off. Most features have additional options too, check them out in the Settings menu."
+  },
+  {
+    "en_us": "With Info Boxes and Custom Bars you can display almost anything you want on your screen using Functions. Join our Discord server to find many community made overlays with {wynntils_command:discord}."
+  },
+  {
+    "en_us": "Got a question? Found a bug or want to suggest something? Join our Discord server and share what you want with {wynntils_command:discord}."
+  },
+  {
+    "en_us": "Remember to check for updates from time to time. Wynntils updates quite frequently and may contain important compatibility fixes."
+  },
+  {
+    "en_us": "Press {keybind:Screenshot Item} on an item to make a screenshot of it."
+  },
+  {
+    "en_us": "Press {keybind:View player's gear} on a player to inspect their gear or perform quick actions."
+  },
+  {
+    "en_us": "Creating a Wynntils Account will let you equip custom cosmetics. To create an account, use {wynntils_command:token}."
+  },
+  {
+    "en_us": "With the Container Scroll feature you can use your mouse scroll to navigate pages in most paginated containers with ease."
+  },
+  {
+    "en_us": "Some features rely on the scoreboard being enabled, make sure you have it enabled by running {toggle_command:sb}."
+  },
+  {
+    "en_us": "Enable autotracking with {toggle_command:autotracking} to ensure your content tracker will always update."
+  },
```

Comment:
Reword these two like the lootrun beacon one otherwise people will toggle them on and off.